x/auth/types/account.go

```diff
@@ -301,6 +336,41 @@ type AccountI interface {
 
 	// Ensure that account implements stringer
 	String() string
+
+	MarshalX() ([]byte, error)
+}
+
+func MarshalAccountX(cdc codec.BinaryMarshaler, acc AccountI) ([]byte, error) {
+	if bacc, ok := acc.(*BaseAccount); ok && bacc.MultisigPubKey == nil {
```

Comment:
```suggestion
	if bacc.MultisigPubKey == nil {
		return acc.MarshalX()
	}
```
Isn't is possible?

Reply:
We don't know if acc is *BaseAccount, so we need to cast it first. AccountI can be one of *BaseAccount, *ModuleAccount or *GenesisAccount.

Reply:
Hence, I'll keep it as is.